src/r2mm/installing/ConflictManagementProviderImpl.ts

```diff
@@ -80,6 +83,15 @@ export default class ConflictManagementProviderImpl extends ConflictManagementPr
                 for (const [key, value] of modFiles.files) {
                     if (value === file) {
                         await FileUtils.ensureDirectory(path.dirname(path.join(profile.getPathOfProfile(), file)));
+
+                        // HACK: Check if UE4SS-settings.ini exists and, if it doesn't, add it. If it does exist, DONT TOUCH.
+                        if (file.toLocaleLowerCase() == "ue4ss-settings.ini") {
+                            if (!await FsProvider.instance.exists(path.join(profile.getPathOfProfile(), file))) {
+                                await FsProvider.instance.copyFile(key, path.join(profile.getPathOfProfile(), file));
+                            }
+
+                            break;
+                        }
```

Comment:
This seems like the wrong location to implement this, there should be code somewhere which ensures all pre-launch configuration is set up right before a launch is attempted & that'd seem like where this should also be placed?

Reply:
Seems like I might be thinking of an EGS specific pre-launch procedure rather than something universal, not that an universal one would be hard to implement. The main point is that I don't think adding more "magic" here is a good idea & finding a place more suitable for it would be better.

Reply:
Managed to refactor this out